frontend/public/components/storage/create-pvc.tsx

```diff
@@ -21,6 +22,18 @@ import {
   getAccessModeForProvisioner,
 } from './shared';
 
+const InfoToolTip = () => {
+  return (
+    <span>
+      <Tooltip
+        position="bottom"
+        content={<div>Only filesystem volume mode is available for cephfs</div>}
+      >
+        <OutlinedQuestionCircleIcon />
+      </Tooltip>
+    </span>
+  );
+};
```

Comment:
No need for `span`. I also miss type and you can do without explicit return 
```suggestion
const InfoToolTip: React.FC = () => (
      <Tooltip
        position="bottom"
        content={<div>Only filesystem volume mode is available for cephfs</div>}
      >
        <OutlinedQuestionCircleIcon />
      </Tooltip>
  );
```

Reply:
I tried with `ocs-storagecluster-cephfs` (provisioner `openshift-storage.cephfs.csi.ceph.com`), but I still could choose Filesystem or Block. The code disables the Block option only for provisioner `ceph.rook.io/block`. So I wonder if the message is correct ?

Reply:
oops index should be 1 :P changed it ! could you give it another shot